backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/slider/internal/BpkRangeSliderImpl.kt

```diff
@@ -96,6 +96,15 @@ internal fun BpkRangeSliderImpl(
                 enabled = enabled,
             )
         },
+        track = {
+            SliderDefaults.Track(
+                colors = sliderColors(),
+                sliderPositions = it,
+                modifier = Modifier.applyIf(upperThumbLabel != null || lowerThumbLabel != null) {
+                    padding(top = LabelOffset)
```

Comment:
offset the track by the label if available. before there was extra spacing below the track as well in the label which is incorrect. the track is always centred otherwise.